Wrapper.php

```diff
@@ -211,6 +211,8 @@ public function stream_metadata($path, $option, $values)
                 $out = chmod($path, $values);
 
                 break;
+            default:
```

Comment:
Something equivalent to https://github.com/hoaproject/Protocol/blob/master/Test/Unit/Wrapper.php#L286-L308.